### PR TITLE
Be 9 be 10 be 11 be 12 be 13 liquebase scripts

### PR DESCRIPTION
**Pull request by AlKaliada**
Create liquibase scriprs for create DB

### CODE REVIEW

**Review by nazfurdychka — APPROVED** [review]

File: src/main/resources/db/changelog/changelog-1.yaml [hunk 1]
```diff
@@ -0,0 +1,729 @@
+databaseChangeLog:  
+  -  changeSet:
+      id: 1
```
Comment (serkushner) [inline src/main/resources/db/changelog/changelog-1.yaml:3]: Could you rename id of liqui script following such template:
20210330.[TiketNumber]-create-interviewer-table

File: src/main/resources/db/changelog/changelog-1.yaml [hunk 2]
```diff
@@ -0,0 +1,729 @@
...
+      author: aleksei
```
Comment (serkushner) [inline src/main/resources/db/changelog/changelog-1.yaml:4]: Could you pls change author name into email

File: src/main/resources/db/changelog/changelog-1.yaml [hunk 3]
```diff
@@ -0,0 +1,729 @@
...
+      changes:
```
Comment (serkushner) [inline src/main/resources/db/changelog/changelog-1.yaml:5]: Pls add precondition before changes section

File: src/main/resources/db/changelog/changelog-1.yaml [hunk 4]
```diff
@@ -0,0 +1,729 @@
...
+            columns:  
+              -  column:  
+                  name:  id  
+                  type:  bigint
+                  autoIncrement:  true  
+                  constraints:  
+                    primaryKey:  true  
+                    nullable:  false  
+              -  column:  
+                  name:  start
```
Comment (serkushner) [inline src/main/resources/db/changelog/changelog-1.yaml:57]: Rename pls into start_date

File: src/main/resources/db/changelog/changelog-1.yaml [hunk 5]
```diff
@@ -0,0 +1,729 @@
...
+                  type: varchar(30)
+              - column:
+                  constraints:
+                    nullable: false
+                  name: subject_id
+                  type: bigint
+              - column:
+                  constraints:
+                    nullable: false
+                  name: start_internship
```
Comment (serkushner) [inline src/main/resources/db/changelog/changelog-1.yaml:239]: pls rename into start_date

File: src/main/resources/db/changelog/changelog-1.yaml [hunk 6]
```diff
@@ -0,0 +1,729 @@
...
+                  type: date                  
+              - column:
+                  constraints:
+                    nullable: false
+                  name: end_internship
+                  type: date
+              - column:
+                  name: start_request
```
Comment (serkushner) [inline src/main/resources/db/changelog/changelog-1.yaml:247]: Pls rename into end_request_date

**Review by serkushner — CHANGES_REQUESTED** [review]

File: src/main/resources/db/changelog/changelog-1.yaml (hunk 1 above)
Comment (AlKaliada) [reply src/main/resources/db/changelog/changelog-1.yaml:3]: fixed

File: src/main/resources/db/changelog/changelog-1.yaml (hunk 2 above)
Comment (AlKaliada) [reply src/main/resources/db/changelog/changelog-1.yaml:4]: fixed

File: src/main/resources/db/changelog/changelog-1.yaml (hunk 5 above)
Comment (AlKaliada) [reply src/main/resources/db/changelog/changelog-1.yaml:239]: fixed

File: src/main/resources/db/changelog/changelog-1.yaml (hunk 6 above)
Comment (AlKaliada) [reply src/main/resources/db/changelog/changelog-1.yaml:247]: fixed

**Review by serkushner — APPROVED** [review]

**Pull request merged**